shared/actions/teams/index.js

```diff
@@ -197,18 +197,17 @@ const _inviteToTeamByPhone = function*(action: Types.InviteToTeamByPhone) {
   })
 
   /* Open SMS */
-  // seitan is 17chars
-  // message sans teamname is 120chars long. Absolute max teamname + descriptor can be is 40chars to fit within 160 sms limit
-  // max length of a teamname is 16chars, + 5|8 chars for descriptor is a max of 24chars for teamDescription
-  let teamDescription
+  // seitan is 18chars
+  // message sans teamname is 118chars. Teamname can be 33 chars before we truncate to 25 and pre-ellipsize
+  let team
   if (teamname.length <= 16) {
-    teamDescription = `${teamname} team`
+    team = `${teamname} team`
+  } else if (teamname.length <= 33) {
```

Comment:
don't we need to check for a period for it to really be a subteam?

Reply:
Root teamnames can only be <= 16chars long, so if it's longer than 16 we can probably assume this is a subteam. But in case of us allowing longer teamnames later, I'll add the period check too